cos-fleetshard-operator-camel/src/main/java/org/bf2/cos/fleetshard/operator/camel/CamelConstants.java

```diff
@@ -16,6 +16,7 @@
     public static final String CONNECTOR_TYPE_SINK = "sink";
 
     public static final String CAMEL_K_PROFILE_OPENSHIFT = "OpenShift";
+    public static final String KAMEL_OPERATOR_ID = "camel.apache.org/operator.id";
```

Comment:
This need to be included among LABELS_TO_TRANSFER 

@astefanutti am I right ?

Reply:
@lburgazzoli it is not required to propagate it to owned resources. It is meant to be added to resources in the `camel.apache.org` API group.

Beware that this is an annotation, not a label.

Reply:
@astefanutti What do mean by owned resources exactly?

Reply:
@shivamG640 the KameletBinding resource is the owner of an Integration, that owns a Deployment, that owns Pod(s). Adding the annotation to the KameletBinding is enough. It gets copied to the owned Integration automatically, and it is not necessary that this annotation be propagated to the owned Deployment.